openstack/03a-IPHC/frag.c

```diff
@@ -324,7 +324,6 @@ void frag_receive(OpenQueueEntry_t *msg) {
 
         if (idmanager_getIsDAGroot() == TRUE) {
             openbridge_receive(msg);
-            return;
```

Comment:
This return and the following one may not affect the running logic of this function. Though, it maybe is easier to understand how the program goes with those returns. Thought?